src/components/datagrid/DataGrid.js

```diff
@@ -86,10 +86,20 @@ export class DataGridComponent extends FormioComponents {
     return {};
   }
 
+
   buildRows(data) {
     this.tbody.innerHTML = '';
     this.rows = [];
     this.components = [];
+
+    // Check minLength validation
+    if (this.component.validate && this.component.validate.hasOwnProperty('minLength') && this.data[this.component.key].length < this.component.validate.minLength) {
+      let toAdd = this.component.validate.minLength - this.data[this.component.key].length;
+      for (let i = 0; i < toAdd; i++) {
+        this.data[this.component.key].push({});
```

Comment:
Why are you doing this? This code looks really suspicious.

Reply:
I did this because we need to have a minLength property. So basically what it those is if you have minLength set will create those fields for you. It's what ngFormio does.